funnel/models/proposal.py

```diff
@@ -93,9 +93,11 @@ class Proposal(BaseScopedIdNameMixin, CoordinatesMixin, db.Model):
     section = db.relationship(ProposalSpaceSection, primaryjoin=section_id == ProposalSpaceSection.id,
         backref="proposals")
     objective = MarkdownColumn('objective', nullable=True)
+    partA = db.synonym('objective')
     session_type = db.Column(db.Unicode(40), nullable=True)
     technical_level = db.Column(db.Unicode(40), nullable=True)
     description = MarkdownColumn('description', nullable=True)
+    partB = db.synonym('description')
```

Comment:
The capitalization of A and B violates our style guide. This should be `part_a` and `part_b` or `part1` and `part2`.